### ISSUE TITLE
cuda extension install nvcc version

### ISSUE THREAD

**Issue by frederickhoffman** [teddykoker/torchsort#50]
What version of nvcc is required to install the cuda extension?

**Comment by teddykoker** [comment]
Yes - on my own machine I am using nvcc release 11.3, with `torch==1.10.2+cu113`. I have tested release nvcc 10.2 with `torch==1.10.2` as well. Let me know if you are running into any issues and I can help!

**Comment by frederickhoffman** [comment]
Hi Teddy,

I've compiled the extension using cuda 10.2 and I've run through the
examples which all work on both CPU and GPU from a .py script. Strangely
the same commands cause segmentation faults for GPU in a jupyter notebook
and also as part of a pytorch lightning training procedure in a python
script. Both work for CPU. Which has left me scratching my head. Any
advice would be greatly appreciated.

Cheers, Fred

**Comment by teddykoker** [comment]
Is there any way you could provide a small sample of your notebook/training script so that I could try and reproduce? Given your description I'm not sure exactly what the issue might be. Does the segmentation fault remain if you remove call to `torchsort`?

**Comment by frederickhoffman** [comment]
Hi Teddy, sure, here is an example of running a gpu torchsort example in a
notebook. This kills the notebook. It works in cpu mode. I'll also provide
a pytorch lightning example, just putting together now. Cheers, Fred

**Comment by teddykoker** [comment]
Fred, could you try to send the examples again? I’m not sure if you attach them in an email if they are uploaded to GitHub

**Comment by frederickhoffman** [comment]
Everything is now fixed after reinstalling everything from scratch. Thanks for all the help Teddy!
